led-adjust.pro

```diff
@@ -0,0 +1,25 @@
+QT += serialport
+QT += core gui
+
+greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
+
+CONFIG += c++17
+
+# You can make your code fail to compile if it uses deprecated APIs.
+# In order to do so, uncomment the following line.
+#DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
+
+SOURCES += \
+    main.cpp \
+    mainwindow.cpp
+
+HEADERS += \
+    mainwindow.h
+
+FORMS += \
+    mainwindow.ui

```

Comment:
Please add all new files you added to the project here. Currently it cant be built with this .pro file.

Reply:
Oops, I forgot to add the new version of this file. It's fixed now.